build/build.ts

```diff
@@ -7,21 +7,30 @@
 
 /// <reference types="bun-types/bun" />
 
-import fs, { write } from 'fs'
-import path from 'path'
 import arg from 'arg'
+import { $, stdout } from 'bun'
 import { build } from 'esbuild'
 import type { Plugin, PluginBuild, BuildOptions } from 'esbuild'
 import * as glob from 'glob'
+import fs from 'fs'
+import path from 'path'
 import { removePrivateFields } from './remove-private-fields'
-import { $, stdout } from 'bun'
+import { validateExports } from './validate-exports'
 
 const args = arg({
   '--watch': Boolean,
 })
 
 const isWatch = args['--watch'] || false
 
+const readJsonExports = (path: string) => JSON.parse(fs.readFileSync(path, 'utf-8')).exports
+
+const [packageJsonExports, jsrJsonExports] = ['./package.json', './jsr.json'].map(readJsonExports)
```

Comment:
I think it also should validate `typesVersions`.

Reply:
thanks, will work on it.

Reply:
@nakasyou Ah, you are right. But perhaps validating `typeVersions` is another issue? We can check whether it's correct without `jsr.json`.

@EdamAme-x What do you think of it?

Reply:
I was able to write the code, but you are right, it may be another problem.　　
I will consider this after this PR is merged.

Reply:
Okay. Let's merge this first.